src/algebra/module/submodule/basic.lean

```diff
@@ -264,13 +264,11 @@ These instances transfer the action by an element `m : M` of a `R`-module `M` wr
 onto the action by an element `s : S` of a submodule `S : submodule R M` such that
 `s +ᵥ a = (s : M) +ᵥ a`.
 
-These instances work particularly well in conjunction with `module.to_add_action`, enabling
```

Comment:
I think this would have been fine if you replaced the bad name with 
`add_group.to_add_action|

Reply:
Oops. Sorry about that. I added it back with the correction.